src/components/v5/shared/SearchSelect/SearchSelect.tsx

```diff
@@ -153,6 +155,19 @@ const SearchSelect = React.forwardRef<HTMLDivElement, SearchSelectProps>(
                 )
               ) : (
                 <>
+                  {isDefaultItemVisible && (
+                    <button
+                      type="button"
+                      className="text-sm md:hover:text-blue-400 flex items-center gap-2 min-h-[3.125rem]
+                      "
+                      onClick={() => onSelect?.(searchValue)}
+                    >
+                      <Avatar />
+                      <span className="max-w-[15.625rem] truncate">
+                        {searchValue}
+                      </span>
+                    </button>
+                  )}
```

Comment:
I would try to find a more meaningful name for this prop. Something like `showSearchValueAsOption` or something like that